client/src/pages/Availability/Availability.tsx

```diff
@@ -0,0 +1,66 @@
+import { SyntheticEvent } from 'react';
+import { Paper, Typography, Box, Button, InputLabel, Grid, List, TextField } from '@material-ui/core';
+import useStyles from './useStyles';
+import { useHistory } from 'react-router-dom';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import DayAvailability from '../../components/DayAvailability/DayAvailability';
+import { updateAvailability } from '../../helpers/APICalls/profile';
+import { Slot } from '../../interface/Profile';
+import useInputState from '../EditProfile/EditProfileForm/useInputState';
+export default function Availability(): JSX.Element {
+  const classes = useStyles();
+  const daysOfWeek = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+  const { updateSnackBarMessage } = useSnackBar();
+  const [rateValue, handleChange] = useInputState('');
```

Comment:
It would be better to have a number as the default state (string is not ideal as this is the price) 😄